Package.swift

```diff
@@ -18,7 +18,14 @@ let package = Package(
         .target(
             name: "YubiKit",
             path: "YubiKit/YubiKit",
-            publicHeadersPath: "SPMHeaderLinks"),
+            publicHeadersPath: "SPMHeaderLinks",
+            linkerSettings: [
+                LinkerSetting.unsafeFlags([
+                    "-weak_framework",
+                    "CoreNFC",
+                    "-weak_framework",
+                    "CryptoTokenKit"
+                ])]),
```

Comment:
Apparently use of `unsafeFlags` makes the package [unable to be used](https://developer.apple.com/documentation/swift_packages/linkersetting/3112682-unsafeflags#discussion) as a dependency [unless the package consumer explicitly points to a git branch](https://bugs.swift.org/browse/SR-11225). Furthermore SwiftPM does not have a mechanism to explicitly weakly link to frameworks. Supposedly Xcode's automatic linking system [should automatically use weak linking](https://forums.swift.org/t/confused-by-unsafe-flags-being-disallowed-in-dependencies/27359/19) when it needs to. I'll mark this as WIP until I can figure out how to get automatic weak linking working.

Reply:
I found a solution to this that doesn't cause any more problems. See the commit message for details.